specs/spec-2-election-ui.md

```diff
@@ -0,0 +1,200 @@
+---
+Spec ID: 2
+Status: Review
+Category: NEAR Social
+Created: 2023-05-25
+Replaces: n/a
+Requires: n/a
+---
+
+# Spec-2: Election UI
+
+## Summary
+
+<!-- One page high-level overview; put details in the specification section and background in the previous section. Should be understandable by a new engineer or reader who isn't already working on the project. Short summary in layman terms. -->
+<!-- List the challenge(s) being solved by this widget. What is the problem? -->
+
+The Election UI provides an interface for people to see the status of current + past NDC Congressional Body elections. This includes the candidates and votes that have been cast. 
+
+Canidates are included in the Election UI after they've been nominated via [Kudos](https://github.com/near-ndc/gwg/blob/main/specs/spec-1-kudos.md) (or a Nominate specific fork of Kudos). After an election the winning candidates are assigned seats in NDC congress.
+
+## Motivation
+
+<!-- Context one needs to know to understand this doc: motivating examples, previous versions and problems, links to related projects/design docs, etc. You should mention related work outside of NEAR if applicable. Note: this is background; do not write about your design or ideas to solve problems here. What's the "historical" context? Why do we need a new project? Identify the problems and the needs. -->
+
+The following is to provide context for the electing the NDC Congressional Bodies, which are the House of Merit, Transparency Commission, and Council of Advisors. Voting is conducted by verified humans in the NDC (denoted as members of the Voting Body).
+
+The overall governance flow:
```

Comment:
should we include in the process a step of candidate presentation (thesis, profile etc...)?

Reply:
MVP won't have user profile pages. Candidate names link to their NS profile, candidate "Nomination" links to Kudos/Nominate.

Reply:
right, I was thinking to add a bit more details to the flow. Or ideally a link to external document where this is discussed.